config/cable.yml

```diff
@@ -0,0 +1,9 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
+  adapter: redis
+  url: redis://localhost:6379/1
```

Comment:
do we have redis?

Reply:
hmm I dont think we do. This was one of the files that got added by the app:update process so I can go back and add redis in now if you think its worth it?